Algorithm.Framework/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NodaTime" version="1.3.4" targetFramework="net461" />
+  <package id="QuantConnect.pythonnet" version="1.0.5.1" targetFramework="net461" />
```

Comment:
Curious why `net461` instead of `net452`, the target framework for the project is correct (v4.5.2)

Reply:
another great catch -- I think this is because when the project was first created it was targeting that version and then I must have added the nuget packages before changing it to 452. Fixed in next push.